src/java.base/share/classes/java/lang/invoke/IndirectVarHandle.java

```diff
@@ -59,6 +64,7 @@
         this.directTarget = target.asDirect();
         this.value = value;
         this.coordinates = coordinates;
+        this.handleMap = this.handleTable = VarHandle.populateMHsJEP383(target, handleFactory);
```

Comment:
`IndirectVarHandle` benegits from lazy initialization of its MethodHandles. OpenJ9's interpreter (`invokevarhandle`) and OpenJ9's VarHandle class do not support lazy initialization. This change will allow OpenJ9 to support JEP383 until OpenJ9 adopts OpenJDK VarHandles.